pontoon/pretranslation/pretranslate.py

```diff
@@ -70,12 +71,23 @@ def get_translations(entity, locale):
 
     # Else fetch from google translate
     elif locale.google_translate_code:
+        gt_input = (
+            entity.string.replace(entity.key, UNTRANSLATABLE_KEY, 1)
+            if entity.resource.format == "ftl"
+            else entity.string
+        )
+
         gt_response = get_google_translate_data(
-            text=entity.string,
+            text=gt_input,
             locale=locale,
         )
 
         if gt_response["status"]:
+            if entity.string != gt_input:
+                gt_response["translation"] = gt_response["translation"].replace(
+                    UNTRANSLATABLE_KEY, entity.key
+                )
```

Comment:
We could log cases where `UNTRANSLATABLE_KEY` doesn't exist in the translated string. But since this is a temporary workaround until we extend Pretranslation beyond internal testing, it's fine if we catch these cases during the review process.